src/components/ArticleList.jsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from "react";
+import ArticleCard from "./ArticleCard";
+import axios from "axios";
+
+const ArticleList = () => {
+const [articles, setArticles] = useState([])
+
+useEffect(()=>{
+    axios.get("https://usmosis-be-nc-news.onrender.com/api/articles/")
```

Comment:
Nice work using Axios. All our API requests should really be in a separate api.js file though. See if you can extract this before the next ticket 👍